### DISCUSSION TITLE
Rename repository to P1P2mqtt

### DISCUSSION THREAD

**Discussion by Timple** [Arnold-n/P1P2MQTT#32] [Ideas]
As this probably better suits the future ideas.

github will automatically redirect the old repo name, not sure if there are other issues involved.

**Comment by gmcmicken** [comment]
I think something like "daikin-mqtt-bridge" would be a little nicer, personally.

**Comment by Arnold-n** [comment]
Thanks @Timple and @gmcmicken, I agree that the name needs a change! I had P1P2MQTT in mind for the new ESP software architecture to replace the current ESP program name P1P2-bridge-esp8266, and P1P2MQTT would be a good name for the overall project too. There may be one issue: the current project is shaped as an Arduino library, and the library would also become P1P2MQTT while the library and (example) program P1P2Monitor do not implement any of the MQTT functionality. But separating into multiple projects also does not make any sense.

Although I like "daikin-mqtt-bridge" as more descriptive, I would prefer not to use any trademark from Daikin in the project name.

The P1P2-ESP-interface needs a better name too, how about P1P2MQTT bridge? Although, one thought here too, the hardware is more generic than P1/P2. It supports any HBS bus with suitable DC voltage, and with the old adapter or some modifications also a powerless HBS bus. In fact, first experiments may soon be done on 3 other HBS buses, including Daikin's F1/F2.

**Reply by gmcmicken** [reply]
Yes I agree, and also I foresee a possibility where even mqtt isn't the only interface to the IP side of the bridge....

How about "homebus-ip-bridge" for the entire thing, and organized into proprietary side and IP side within the repo?

**Reply by Arnold-n** [reply]
Not so sure: the bus is HBS, for Home Bus System, and it beats me why this name was selected. It is far too generic and thus confusing. And I fear ip is also rather generic while I think MQTT (and MQTT discovery for HA) will likely remain dominant. I am thinking of adding a web server but that is too generic to use in a name.. And telnet is only for raw access.

We can sleep over it, I think a good name to change project name is when the new data architecture becomes public for the ESP code.

**Reply by gmcmicken** [reply]
Sure. Keep in mind too when people search GitHub it uses the description as well, the repo name IMO shouldn't necessarily be too specific and many repos are named friendly or memorable coined terms or otherwise. 

My idea is that the repo shouldn't be named anything so specific as to being misleading in the future, necessitating the conversation we're having today about changing the name.

**Comment by Arnold-n** [comment]
Finally renamed to P1P2MQTT, given current focus on MQTT and HA.
